proto/insonmnia.proto

```diff
@@ -132,18 +110,11 @@ message TaskLogsChunk {
     bytes data = 1;
 }
 
-message DiscoverHubRequest {
-    string endpoint = 1;
-}
-
 message TaskResourceRequirements {
     // CPUCores specifies the number of CPU cores required.
     uint64 CPUCores = 1;
     // MaxMemory specifies the maximum memory in bytes required.
     int64 maxMemory = 2;
-    // GPUCount Describes whether a task requires GPU support.
-    // todo: completely useless in that representation.
-    GPUCount GPUSupport = 3;
```

Comment:
Looks like it should be changed to something else instead of deletion, but ok for now